### PR TITLE
ICSAAS-0000 update naming of auto-complete feature related classes, v…

### PR DESCRIPTION
**Pull request by igorsimovski**
…ariables, labels...

### CODE REVIEW

File: ui.frontend/src/main/webpack/site/js/types/searchOptions.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { TabConfig } from '../components/searchTabs'
-import { SearchCallbacks } from './callbacks'
+import {TabConfig} from '../components/searchTabs'
```
Comment (kevinolivar) [inline ui.frontend/src/main/webpack/site/js/types/searchOptions.ts:1]: @igorsimovski we have apparently different formatting.

I am using prettier. May want to align this at some points with backend devs :p

File: ui.frontend/src/main/webpack/site/js/components/searchInput.ts [hunk 2]
```diff
@@ -175,13 +175,13 @@ const debouncedSearch = (autoSuggestionDebounceTime: number) =>
...
-  searchFieldPlaceholderText,
-  autocompleteUrl,
-  autocompleteTriggerThreshold,
-  autoSuggestionDebounceTime = 500,
-  searchContainer,
-}: SearchInputOptions): HTMLInputElement => {
+                            id,
+                            searchFieldPlaceholderText,
+                            autoCompleteUrl,
+                            autoCompleteTriggerThreshold: autoCompleteTriggerThreshold,
```
Comment (kevinolivar) [inline ui.frontend/src/main/webpack/site/js/components/searchInput.ts:181]: You can keep autoCompleteTriggerThreshold with the ": autoCompleteTriggerThreshold" it is the same, just shorter.

File: README.md [hunk 3]
```diff
@@ -191,14 +191,15 @@ Configurations are split in OSGi and Context-Aware.
...
 | Enable Auto Suggest              | enableAutoSuggest            | Flag that enables auto suggest feature in the search component. Defaults to `true`                                         |          |
-| Enable Autocomplete              | enableAutocomplete           | Flag that enables autocomplete feature in the search input. Defaults to `true`                                             |          |
-| Autocomplete Trigger Threshold   | autocompleteTriggerThreshold | The minimum number of search input characters required for displaying autocomplete options.                                |          |
-| Autocomplete options Max Limit   | autocompleteOptionsMax       | The maximum number of autocomplete options displayed.                                                                      |          |
+| Enable Auto Complete             | enableAutoComplete           | Flag that enables auto-complete feature in the search input. Defaults to `true`                                            |          |
```
Comment (kevinolivar) [inline README.md:200]: **Thought**
auto complete / auto-complete? :D 

Is auto-complete used as an adjective?

This English naming can be mental! XD

Comment (nhirrle) [reply README.md:200]: Important is that we are consitent. Let's use "Autocomplete" everywhere as in wikipedia https://en.wikipedia.org/wiki/Autocomplete

File: ui.frontend/src/main/webpack/site/js/types/searchOptions.ts (hunk 1 above)
Comment (igorsimovski) [reply ui.frontend/src/main/webpack/site/js/types/searchOptions.ts:1]: can you supply me with the formatter, so i can fix the formatting for FE files?

File: README.md (hunk 3 above)
Comment (igorsimovski) [reply README.md:200]: > Important is that we are consitent. Let's use "Autocomplete" everywhere as in wikipedia https://en.wikipedia.org/wiki/Autocomplete . so no dashes, spaces nor camelcase

oh, so the PR is obsolete then :D
will decline it

**Pull request closed without merging**